scope_jason.py

```diff
@@ -0,0 +1,120 @@
+import pandas as pd
+import pdb
+import json
+from penquins import Kowalski
+import numpy as np
+
+def get_highscoring_objects(otype='vnv',condition="$or",
+    limit=0.9,limit_dnn=None,limit_xgb=None,):
+
+    if limit_dnn == None:
+        limit_dnn = limit
+    if limit_xgb == None:
+        limit_xgb = limit
+
+    ### example
+    q = {"query_type": "find",
+             "query": {
+                 "catalog": "ZTF_source_classifications_DR5",
+                 "filter": { condition : [{'%s_xgb' %otype: { '$gt': limit_xgb }},
+                                    {'%s_dnn' %otype: { '$gt': limit_dnn }}],
+                           },
+                 "projection": {}
+             },
+             "kwargs": {     }
+             }
+
+    r = G.query(q)
+
+    return pd.DataFrame(r['data'])
+
+def get_stats(ids):
+    qs = [
+           {"query_type": "find",
+             "query": {
+                 "catalog": "ZTF_source_features_DR5",
+                 "filter": {'_id': i
+                            },
+                 "projection": {}
+             },
+             "kwargs": {     }
+             }
+        for i in ids
+    ]
+    rs = G.batch_query(qs, n_treads=32)
+
+    return pd.DataFrame([s['data'][0] for s in rs])
+
+# load dataset
+trainingset = pd.read_csv('dataset.d15.csv')
```

Comment:
Make this file path a command line argument.

Reply:
could you explain more how to do that?

Reply:
https://realpython.com/python-command-line-arguments/